### PR TITLE
RE2022-340 GTDB special casing docs

### PR DESCRIPTION
**Pull request by dauglyon**

### CODE REVIEW

File: src/features/collections/README.md
Comment (MrCreosote) [inline src/features/collections/README.md:1]: isn't there special casing for the plots as well? I thought there were some other cases but they're not coming to mind

Comment (dauglyon) [reply src/features/collections/README.md:1]: Accidentally missed one related to the plots, adding it. But that seems to be it.

Comment (dauglyon) [reply src/features/collections/README.md:1]: added

Comment (MrCreosote) [reply src/features/collections/README.md:1]: I'm talking abour special casing the axes of the plots, not just special casing them for gtdb. e.g. have the backend specify the plots

Comment (dauglyon) [reply src/features/collections/README.md:1]: Oh, at the moment the plots are "hard coded" with column names (as component parameters) to use for the axes. IIRC we didn't decide to specify those things on the server. Those component parameters are then special cased for GTDB as it has different column names.

Comment (dauglyon) [reply src/features/collections/README.md:1]: Maybe it would be good to also document anywhere column names are used "hardcoded"?

Comment (dauglyon) [reply src/features/collections/README.md:1]: I can think of two off the top of my head: plots, and the classification search box

Comment (MrCreosote) [reply src/features/collections/README.md:1]: Yeah, hardcoded stuff = special casing to my mind

File: package.json [hunk 1]
```diff
@@ -128,7 +128,7 @@
       "lint": {},
       "lint-list-item-indent": "mixed",
       "lint-maximum-line-length": 80,
-      "lint-no-missing-blank-lines": true,
+      "lint-no-missing-blank-lines": false,
```
Comment (dauglyon) [inline package.json:131]: this being set to true returns a false positive lint error for nested markdown lists

File: src/features/collections/README.md
Comment (dauglyon) [reply src/features/collections/README.md:1]: done :)

File: package.json (hunk 1 above)
Comment (dauglyon) [reply package.json:131]: @dakotablair you enabled this, so you may want to review its disabling

**Review by MrCreosote — COMMENTED** [review]
LGTM but @dakotablair should give the final approval

File: package.json (hunk 1 above)
Comment (dauglyon) [reply package.json:131]: Hmm, changing the spacing didn't work for me (it then complained about incorrect indent levels)

**Pull request merged**